### ISSUE TITLE
Allow overriding configured model

### ISSUE THREAD

**Issue by apetresc** [rikhuijzer/ata#31]
Sometimes you want to drop into GPT-4 for a particularly tough session, but normally keep the cheaper GPT-3.5. It's annoying to edit the config file every time I want to switch. It would be nice to just be able to do:

```
$ ata --model gpt-4
```

**Comment by rikhuijzer** [comment]
Yes, that sounds like a good idea. Would a keyboard shortcut for switching inside the running application be even better?

**Comment by apetresc** [comment]
Maybe, but since switching the model necessitates restarting the conversation anyway, I wouldn't mind if I just had to restart ata anyway.

**Comment by marioseixas** [comment]
> Yes, that sounds like a good idea. Would a keyboard shortcut for switching inside the running application be even better?

hell yeah!

ATA for the win

for long conversations that need to keep context I'm using https://github.com/felixbade/assistant
he made the shortcut ctrl+M to change the models, very nice

**Comment by jmonroynieto** [comment]
As a new user, this was one of the things that I looked for first.

**Comment by ctrlcctrlv** [comment]
I implemented this via environment variables in ctrlcctrlv#1
